packages/esm-ohri-core-app/src/index.ts

```diff
@@ -51,9 +51,9 @@ function setupOpenMRS() {
         offline: true,
       },
       {
-        id: 'ohri-all-patients-list',
+        id: 'ohri-home',
         slot: 'ohri-home-dashboard-slot',
-        load: getAsyncLifecycle(() => import('./components/all-patients-list/patient-list.component'), {
+        load: getAsyncLifecycle(() => import('./ohri-dashboard/home/ohri-home.component'), {
```

Comment:
What you are actually doing here is: Plugging the "OhriHome" component into the "Ohri Home Dashboard" which introduces some confusion to the tree. You only want to plug the patient list into the "Ohri Home Dashboard". Simply rename the "OhriHome" component at path: `/ohri-dashboard/home/ohri-home.component` to something like "AllPatientsList"

Reply:
That's okay...let me undo this and use the previous setup